### ISSUE TITLE
Can't Use Refresh Token

### ISSUE THREAD

**Issue by planecore** [tdorssers/TeslaPy#55]
Hey! I'd like to use this library with a refresh token I supply.

I saw this example in the README, but it still brings up the browser auth. Do you have any idea why?

```
import teslapy
with teslapy.Tesla('my@email.com') as tesla:
    if not tesla.authorized:
        tesla.refresh_token(refresh_token='my-refresh-token')
```

**Comment by tdorssers** [comment]
This issue is fixed in aee1228. I will upload a new version including this fix to pypi shortly.

**Comment by tdorssers** [comment]
TeslaPy 2.4.0 has been uploaded to [pypi](https://pypi.org/project/TeslaPy/).
